packages/gatsby/src/commands/build-html.ts

```diff
@@ -35,15 +41,19 @@ const doBuildRenderer = async (program, webpackConfig) => {
   return outputFile
 }
 
-const buildRenderer = async (program, stage, { parentSpan }) => {
+const buildRenderer = async (
+  program: IProgram,
+  stage: BuildHTMLStage,
+  parentSpan: IActivity
+): Promise<string> => {
   const { directory } = program
   const config = await webpackConfig(program, directory, stage, null, {
     parentSpan,
```

Comment:
As per the previous comment, this has changed from the previous code.

Reply:
I don't actually know the context around this, but I see it everywhere. Just wrapping in an object doesn't change any references, so we are still always using the same instance all around. I'm going to start removing these blocks as I can.

Reply:
asking in this thread. https://gatsbyjs.slack.com/archives/CA1GW1HNU/p1579121474098900